### PR TITLE
Fix editor error and added single line option

### PR DESCRIPTION
**Pull request by JonathanSouthern**
**MADE FOR:** https://github.com/workbase-inc/workbase-mono/pull/47

Changes:
- Fix editor bug
- Adjusted Autocomplete
- Allow single line editor now

### CODE REVIEW

File: src/MUIRichTextEditor.tsx [hunk 1]
```diff
@@ -201,7 +202,7 @@ const styleRenderMap: DraftStyleMap = {
 }
 
 const { hasCommandModifier } = KeyBindingUtil
-const autocompleteMinSearchCharCount = 2
+const autocompleteMinSearchCharCount = 0
```
Comment (ed-workbase) [inline src/MUIRichTextEditor.tsx]: we should turn this into a prop

File: src/MUIRichTextEditor.tsx [hunk 2]
```diff
@@ -280,6 +288,13 @@ const MUIRichTextEditor: RefForwardingComponent<TMUIRichTextEditorRef, IMUIRichT
         start: 0,
         end: 0
     })
+    const autocompleteOptions = React.useMemo(()=>{
```
Comment (ed-workbase) [inline src/MUIRichTextEditor.tsx]: can we rename this to `memoedAutocompleteItems`

File: src/MUIRichTextEditor.tsx [hunk 3]
```diff
@@ -1010,6 +1028,7 @@ const MUIRichTextEditor: RefForwardingComponent<TMUIRichTextEditorRef, IMUIRichT
                     setSelectedIndex(limit - 1)
                 }
                 return "mui-autocomplete-navigate"
+            //TODO: Fix 'Enter' key event. Function isn't currently recieving 'enter' key
```
Comment (ed-workbase) [inline src/MUIRichTextEditor.tsx]: there was a merge from gitstart that will have changed this behaviour

**Review by ed-workbase — COMMENTED** [review]
comments inline

File: src/MUIRichTextEditor.tsx (hunk 2 above)
Comment (JonathanSouthern) [reply src/MUIRichTextEditor.tsx]: Resolved

File: src/MUIRichTextEditor.tsx (hunk 3 above)
Comment (JonathanSouthern) [reply src/MUIRichTextEditor.tsx]: Resolved/Fixed

File: src/MUIRichTextEditor.tsx (hunk 1 above)
Comment (JonathanSouthern) [reply src/MUIRichTextEditor.tsx]: Resolved/Fixed

**Review by ed-workbase — APPROVED** [review]

**Pull request merged**